### DISCUSSION TITLE
posts/b42607e4e3c6/

### DISCUSSION THREAD

**Discussion by giscus** [boringhex-top/blog.boringhex.top#2] [Announcements]
# posts/b42607e4e3c6/

踟蹰了太长时间，差点就放弃，但是心中还是觉得技术人应该写点什么，留下点什么，所以终于还是入坑了。 从弃坑到入坑 第一次产生写博客的想法应该还是大学时，当时经常逛论坛，翻看技术博客，主要是老外的，当时真的是啥啥芯片都是老外的，资料也就基本英文。看人家写的博客就觉得这帮人怎么这么闲，每天干完工作，还有时间写博客、发文章，甚至拍视频，做技术分享交流，老外的文章质量真的很高，甚至比国内某些教材写的都好，所

http://localhost:4000/posts/b42607e4e3c6/

**Comment by pengwon** [comment]
测试一下
